packages/web-components/src/components/callout-quote/callout-quote.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './callout-quote.scss';
+import '../callout/callout';
+import '../quote/quote';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Callout Data.
+ *
+ * @element dds-callout-data
+ */
+@customElement(`${ddsPrefix}-callout-quote`)
+class DDSCalloutQuote extends LitElement {
+  render() {
+    return html`
+      <dds-callout>
+        <dds-quote inverse>
```

Comment:
As discussed in #4122, would we be able to do theme zoning with CSS custom properties instead of relying on `inverse` attribute here?

Reply:
FYI, this can be done by:

```scss
:host(#{$dds-prefix}-callout-quote) #{$dds-prefix}-quote {
  @include carbon--theme($carbon--theme--g100, true);

  // The original quote text color inherits from body.
  // Re-defines the color to apply the theme zoning.
  color: $text-01;
}
```

An alternate approach is making `<dds-callout-quote>` inherit `<dds-quote>`. It can be done along with #4170.